spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/Sanitizer.java

```diff
@@ -55,8 +55,9 @@
 	private static final Set<String> URI_USERINFO_KEYS = new LinkedHashSet<>(
 			Arrays.asList("uri", "uris", "url", "urls", "address", "addresses"));
 
-	private static final Pattern URI_USERINFO_PATTERN = Pattern
```

Comment:
Hi, the reason for the newline here is because I found that the original expression:

> `^\\[?[A-Za-z][A-Za-z0-9\\+\\.\\-]+://.+:(.*)@.+$`

Maybe it can be simplified to:

> `^\[?[A-Za-z][A-Za-z0-9+.\-]+://.+:(.*)@.+$`

(`[A-Za-z0-9\\+\\.\\-]` changed to `[A-Za-z0-9+.\-]`)

Reply:
Ah, i didn't see the change. Then it's fine :)